_content/articles/opening-debate-in-the-milindapanha_analayo.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "The Opening Debate in the Milindapañha"
+authors:
+  - analayo
+external_url: "https://www.buddhismuskunde.uni-hamburg.de/pdf/5-personen/analayo/openingdebate.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1q1u1XmUL7yaxFRSuKsk3jwc4eA5y-TPt/view?usp=sharing"
+course: dialogue
+tags:
+  - anatta
+  - theravada-roots
+year: 2021
+journal: "Sri Lanka International Journal of Buddhist Studies"
+publisher:"Sri Lanka International Buddhist Academy" 
```

Comment:
This broke the build. For future reference, there must be a space after the colon.